src/Translator.php

```diff
@@ -43,6 +52,24 @@ public function translate($locale = null, $fallback = true)
             $this->translateAttribute($attribute, $locale, $fallback);
         }
 
+        // Translate Relations
+        foreach ($this->model->getRelations() as $key => $relation) {
+            if ((new Voyager)->translatable($relation)) {
```

Comment:
First of all you are a life save I came here looking for this as I was trying to do the same thing, the only problem for me with your solution that i didn't want the translations to be loaded separately after the translate did the work, so I added this peace of code to remove the translations relations 

if ($key == "translations") {
                unset($this[$key]);
                continue;
  }

Reply:
```suggestion
            if ((new Voyager)->translatable($relation)) {
                        if ($key == "translations") {
                                    unset($this[$key]);
                                    continue;
                        }
```

Reply:
Thank you so much for your modification, that's really good too, agreed!